lib/surface/compiler/eex_engine.ex

```diff
@@ -613,6 +614,8 @@ defmodule Surface.Compiler.EExEngine do
          %AST.Attribute{name: name, type: type, value: [%AST.Text{value: value}]}
          | attributes
        ])
+       # TODO: [Type] Should we have something like Surface.TypeHandler.literal_to_html(...) which
```

Comment:
We should be able to rely on `Surface.attr` here (or directly on TypeHandler.attr_to_value`) instead of inserting the result directly. The one thing I'm unsure about is because that uses `Phoenix.HTML.raw`, we may have to unwrap that. If the unwraping became an issue, we could go for the `literal_to_html` route and just have that return the same thing as attr_to_html (but without the call to raw).

An alternative is to remove that call to `Phoenix.HTML.raw` in the type handler, and just call that in `Surface.attr` on the result of `attr_to_html`

Reply:
Yeah, I think we'll need to make a few experiments to see which approach would be more suitable.